examples/cached_alias_config.py

```diff
@@ -33,6 +33,7 @@
     assert await cache.get("key") == "value"
     assert isinstance(cache, SimpleMemoryCache)
     assert isinstance(cache.serializer, DefaultSerializer)
+    await cache.close()
```

Comment:
This is not ideal. `caches.get('default'` returns a singleton identified by 'default'. User may want to use this cache in another function.

Global aliased caches should be closed when the program exits